keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -699,6 +703,24 @@ The logic in
 not likely will need changes, but needs to be tested with the sidecar
 containers.
 
+#### Resource calculation and version skew
+
+In case of a version skew between scheduler and kubeler, or in cases when
+scheduler and kubelet has a different value set for the `Sidecar` feature gate,
+calculation of resources required for a Pod will differ between the scheduler
+and a kubelet when the sidecar container created.
+
+In case when scheduler "knows" about the sidecar and kubelet doesn't, there
+unlikely be any issues. Scheduler will calculate resources usage for a Pod that
+will be equal or more than kubelet will require to run the Pod. So there will be
+no overbooking.
+
+If scheduler has the `Sidecar` feature gate disabled, the Pod that has a Sidecar
```

Comment:
if there is a Pod that was scheduled before it was disabled, we need to keep calculating the resources. Currently we will fallback to the old way of calculating resources which may create a situation of overbooking nodes that has this feature gate enabled.

One option we have is to ignore the feature gate and always calculate resources properly accounting for the sidecar. This will eliminate cases of overbooking. wdyt?

Reply:
For sure for scheduled pods we should ignore the feature gate.

Add a link here to the "downgrade strategy" section, which clarifies what kubelet will do with a disabled feature gate, unless I'm interpreting it incorrectly?

I think it's safer that scheduler doesn't schedule pods with sidecars when the feature gate is disabled. The point of the feature gate is to "stop bleeding". So if there were unscheduled pods and the admin later disables the feature, the pods should remain unschedulable as opposed to treating the sidecars as regular init containers.

Reply:
> For sure for scheduled pods we should ignore the feature gate.

+1
@SergeyKanzhelev  - can you please add this?

Reply:
Added